app.js

```diff
@@ -0,0 +1,127 @@
+const statusDisplay = document.querySelector('.game--status');
+const board = document.querySelector('.board');
+const form = document.getElementById('form');
+const alert = document.querySelector('.alert');
+let gameActive = true;
+
+function Player(name, symbol) {
+  this.name = name;
+  this.symbol = symbol;
+}
+
+const playerOne = new Player('', 'X');
+const playerTwo = new Player('', 'O');
+
+
+const currentPlayerTurn = (cp) => `It's ${cp}'s turn`;
+
+let gameState = ['', '', '', '', '', '', '', '', ''];
+
+/* eslint no-use-before-define: ["error", { "functions": false }] */
+
+function handleCellClick(clickedCellEvent) {
+  const clickedCell = clickedCellEvent.target;
```

Comment:
- [ ] It has a function with the only responsibility of rendering the contents of the gameboard array to the webpage. 

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/render_requirement.png)

handleCellClick is doing too much, it listens for cell click, and then with the help of `handleCellPlayed` will update the board array, and then updating the dom (game logic is not separated from dom manipulation)